internal/store/err.go

```diff
@@ -23,6 +23,8 @@ var (
 	ErrGitNothingToCommit = fmt.Errorf("git has nothing to commit")
 	// ErrEmptySecret is returned if a secret exists but has no content.
 	ErrEmptySecret = fmt.Errorf("empty secret. see https://go.gopass.pw/faq#empty-secret")
+	// ErrMeaninglessWrite is returned if a secret is overwritten with its current (cyphertext) content
```

Comment:
For the sake of consistency, please change `cyphertext` to `ciphertext`. I think that's how we usually call it in the project.